src/components/Signup.jsx

```diff
@@ -16,7 +16,7 @@ const Signup = props => {
         props.changeAuthMessage(`Welcome! ${userDatas.data.data.email}`);
       })
       .catch(error => {
-        if (error.message == "Request failed with status code 422") {
+        if (error.message === "Request failed with status code 422") {
```

Comment:
I must have missed this in the previous PR, why are you not using the real error response from the backend?

Reply:
I also see that you are using a `then, catch` here.  We try to move away from old syntax as much as possible. Especially when we use async methods so much in this project.

Reply:
this is the syntax that comes with j-tockauth

Reply:
Isnt that the error message coming from devise?

Reply:
https://github.com/CraftAcademy/ca-cooper-backend/blob/updated_dec_2019/spec/requests/api/v1/registrations_spec.rb

here you have the error messages that devise sends back

Reply:
sorted